src/protocols/rdp/rdp_fs.c

```diff
@@ -38,7 +38,11 @@
 
 #include <guacamole/pool.h>
 
-guac_rdp_fs* guac_rdp_fs_alloc(guac_client* client, const char* drive_path) {
+guac_rdp_fs* guac_rdp_fs_alloc(guac_client* client, const char* drive_path, int create_drive_path) {
+
+	if(create_drive_path) {
```

Comment:
Please:

1. Include a comment describing what is happening here (see most of the rest of the guac source).
2. Include a single space between the "if" and the "(".

We also normally prefer to omit the braces when only one line follows, but there is a different issue on the next line that will make this not matter.